package.json

```diff
@@ -143,6 +144,7 @@
     "wordwrap": "1.0.0"
   },
   "devDependencies": {
+    "@babel/plugin-external-helpers": "7.0.0",
```

Comment:
@drewvolz Why did we add this package, if our babel config is just `metro-react-native-babel-preset`?

Reply:
(I see that the article says to install it, but I don't like installing things that we don't directly use.)

Reply:
As suggested [here](https://github.com/facebook/react-native/issues/21219#issue-362115854) and fixed [here](https://github.com/facebook/metro/commit/48fa696ca6fc38b0bdd8c86feade620f3c4d0a34)

> Metro depends on a package that it does not specify in its package.json. It is causing the following error:
>
> Loading dependency graph, done.
error: bundling failed: Error: Cannot find module '@babel/plugin-external-helpers'
[...]

Reply:
But I haven't checked to see if it landed / we're picking it up.

Reply:
FWIW, https://github.com/facebook/metro/commit/48fa696ca6fc38b0bdd8c86feade620f3c4d0a34 says it landed in 0.45.5 and is still there in 0.48.1, according to the github tags list, and our Yarn lockfile / `npm info react-native` says that we're depending directly on metro@0.48.1 as of this branch.

Thus, I think we can remove this direct dependency.

Reply:
Great, thanks for looking.